README.md

```diff
@@ -121,53 +119,68 @@ loadModules(['esri/map'], options)
+
+#### When you load the script
+
+The easiest way to do that is to pass the `css` option to `loadModules()` or `loadScript()`:
 
 ```js
 import { loadModules } from 'esri-loader';
 
-// lazy load the CSS before loading the modules
-const options = {
-  css: 'https://js.arcgis.com/4.10/esri/css/main.css'
-};
+// before loading the modules for the first time, 
+// also lazy load the CSS for the version of 
+// the script that you're loading from the CDN 
+const options = { css: true };
 
 loadModules(['esri/views/MapView', 'esri/WebMap'], options)
   .then(([MapView, WebMap]) => {
-    // you can safely create a map now that the styles are loaded
+    // the styles, script, and modules have all been loaded (in that order)
   });
 ```
 
-Alternatively, you can use the provided `loadCss(url)` function to control when the ArcGIS styles are loaded. For example:
+Passing `css: true` does not work when loading the script using the `url` option. In that case you'll need to pass the URL to the styles like: `css: 'http://server/path/to/esri/css/main.css'`.
+
+When passing the `css` option to `loadModules()` it actually passes it to `loadScript()`, So you can also use the same values (either `true` or stylesheet URL) when you call `loadScript()` directly.
+
+#### Using loadCss()
+
+Alternatively, you can use the provided `loadCss()` function to load the ArcGIS styles at any point in your application's life cycle. For example:
 
 ```js
-// load esri styles for version 4.x using loadCss
 import { loadCss } from 'esri-loader';
-loadCss('https://js.arcgis.com/4.10/esri/css/main.css');
-```
 
-See below for information on how to [override ArcGIS styles](#overriding-arcgis-styles) that you've lazy loaded with `loadModules()` or `loadCss()`.
+// by default loadCss() loads styles for the latest 4.x version
+loadCss();
 
-Finally, you can manually load them by more traditional means such as adding `<link>` tags to your HTML, or `@import` statements to your CSS. For example:
+// or for a specific CDN version
+loadCss('3.27');
 
-```html
-<!-- load esri styles for version 4.x via an html link tag -->
-<link rel="stylesheet" href="https://js.arcgis.com/4.10/esri/css/main.css">
+// or a from specific URL, like a locally hosted version
+loadCss('http://server/path/to/esri/css/main.css');
 ```
 
-or:
+See below for information on how to [override ArcGIS styles](#overriding-arcgis-styles) that you've lazy loaded with `loadModules()` or `loadCss()`.
 
-```css
-/* load esri styles for version 3.x via css import */
-@import url('https://js.arcgis.com/3.27/esri/css/esri.css');
-```
+#### Using traditional means
+
+Of course, you don't need to use esri-loader to load the styles. See the [ArcGIS API for JavaScript documentation](https://developers.arcgis.com/javascript/) for more information on how to load the ArcGIS styles by more traditional means such as adding `<link>` tags to your HTML, or `@import` statements to your CSS.
```

Comment:
if you aren't going to link directly to an article that provides more information about loading CSS in the JSAPI, probably better to skip it entirely.